electrum/gui/qt/main_window.py

```diff
@@ -269,6 +271,19 @@ def on_version_received(v):
             self._update_check_thread.checked.connect(on_version_received)
             self._update_check_thread.start()
 
+        self.wallet_info_notification_buttons = None
+        if EmailNotificationWallet.is_subscribable(self.wallet):
+            self._wallet_info_notification = WalletInfoNotifications(self, self.config, self.wallet, self.app)
+
+            def wallet_info_notification_buttons(dialog):
+                button = QPushButton()
+                self._wallet_info_notification.dialog = dialog
+                self._wallet_info_notification.sub_unsub_button = button
+                self._wallet_info_notification.sync_sub_unsub_button()
+                return Buttons(button, CloseButton(dialog))
+
+            self.wallet_info_notification_buttons = wallet_info_notification_buttons
```

Comment:
Wouldnt it be better with lambda?

Reply:
Do you mean assign lambda expression into variable, like `self.wallet_info_notification_buttons = lambda ...`?

Reply:
Yup

Reply:
It's considered as anti-pattern so better not applied it :)